### DISCUSSION TITLE
lucit-backtesting

### DISCUSSION THREAD

**Discussion by oliver-zehentleitner** [oliver-zehentleitner/backtesting.py-oz#1] [Announcements]
### 1.0.0
(2024-06-06)

Takeover of maintenance by LUCIT. The aim is an easy-to-install and functioning system.

* Fix 'ValueError: Length of values (2) does not match length of index (1)' [#649](https://github.com/kernc/backtesting.py/issues/649)
* Fix 'Plotting does not work and raises an error with Bokeh' [#1069](https://github.com/kernc/backtesting.py/issues/1069)

**Full Changelog**: https://lucit-backtesting.docs.lucit.tech/changelog.html
**Compare**: https://github.com/LUCIT-Systems-and-Development/lucit-backtesting/compare/0.3.3...1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/LUCIT-Systems-and-Development/lucit-backtesting/releases/tag/1.0.0'>lucit-backtesting</a>.</em>

**Comment by jmarsh411** [comment]
You're a hero for picking up maintenance. I'll be submitting PRs for any bugfixes I find while using the library. Thank you!

**Reply by oliver-zehentleitner** [reply]
Hello  @jmarsh411!

Thank you very much! That would be great, we will only be able to do this as a community!
